src/js/sequences/components/IsolateSequences.tsx

```diff
@@ -35,7 +45,12 @@ export default function IsolateSequences({ activeIsolate, otuId }) {
     const sequences = sortSequencesBySegment(activeIsolate.sequences, otu.schema);
 
     const Sequence = data_type === "barcode" ? BarcodeSequence : GenomeSequence;
-    let sequenceComponents = map(sequences, sequence => <Sequence key={sequence.id} {...sequence} />);
+    let sequenceComponents: JSX.Element | JSX.Element[] = map(sequences, sequence => (
```

Comment:
Shouldn't this always be an array of JSX elements, just in some cases an empty or array of one?

Reply:
@raastinn Make changes as necessary to simplify the typing.